### PR TITLE
ride list, ride details, settings page

### PR DESCRIPTION
**Pull request by mavillas**

### CODE REVIEW

File: new_prototype/css/ride_details.css [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+section#forms {
+    width: 375px;
+    height: 143px;
+    background-color: #f3f3f3;
+}
+
+section#list {
+    text-align: left;
+    width: 308px;
+    margin: 0 auto;
```
Comment (takumatakuma) [inline new_prototype/css/ride_details.css]: maybe good to add a little margin on the top?
screen shot 2016-05-15 at 6 11 08 pm

**Comment by takumatakuma** [comment]
looks like the icons aren't centered. Also, is it ok to use a different font for the labels?
screen shot 2016-05-15 at 6 19 12 pm

File: new_prototype/ride_list.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+<body>
+    <section id="page">
+        <header>
+            <img src="assets/status_bar.svg" alt="status bar">
+            <h1>Ride List</h1>
+        </header>
+        <section id="content">
+            <section id="forms">
+                <form>
```
Comment (takumatakuma) [inline new_prototype/ride_list.html]: I think it's better to have a submit button for this form

**Comment by takumatakuma** [comment]
What is rider.html for? Does this let a rider to create a ride request?

**Comment by takumatakuma** [comment]
I was thinking the following user flows:

For a driver to create a ride:
Top -> NewRide(driver.html) -> submit

For a rider to find a ride:
Top -> RideList(ride_list.html) -> RideDetail(ride_details.html) -> "Ride this ride"

I would prefer to stay in this model for beta just to keep it as simple as possible. Also requesting feature needs a different logic so it takes extra time to complete.

**Comment by takumatakuma** [comment]
other than the above lgtm

**Comment by mavillas** [comment]
1) Will add margin-top to the ride list. Didnt know I forgot to.

2) Not sure why that font is showing up on yours. The icons+text are part of an image (svg) so I dont think the font should be changing. I can make it a png file and maybe thatll fix it
 
3) Which button should be a submit form?

4) Not sure about that user flow for right now. I’ve talked with Glenda about the designs and she thinks they work well and is expecting to see them in the prototype. But for now if you cant implement the actual rider.html page then maybe we can work something out

File: new_prototype/css/ride_details.css (hunk 1 above)
Comment (takumatakuma) [inline new_prototype/css/ride_details.css]: line note

**Comment by takumatakuma** [comment]
> 2) Not sure why that font is showing up on yours. The icons+text are part of an image (svg) so I dont think the font should be changing. I can make it a png file and maybe thatll fix it

Yes the font is a part of the icon. I mean the font in the icon is different from the fonts used for other texts on the pages, and I am not sure you are intentionally doing that

**Comment by takumatakuma** [comment]
> 4) Not sure about that user flow for right now. I’ve talked with Glenda about the designs and she thinks they work well and is expecting to see them in the prototype. But for now if you cant implement the actual rider.html page then maybe we can work something out

We can absolutely keep the design but I just wanted to make sure we are sharing the necessary functions for the beta.

**Pull request merged**